### DISCUSSION TITLE
How to always hide notification with the following content

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#2624] [General]
I setup an autocommand that toggles capslock off when on insert mode. But this notification is fired almost everytime.
How can I hide it permanently

![Screenshot_20240228_185534](https://github.com/LazyVim/LazyVim/assets/8222059/e6bbe02b-6c73-49e1-bb57-c6368960803b)

**Comment by rubiin** [comment]
@dpetka2001 can you help me with this

**Reply by dpetka2001** [reply]
This seems to be working
```lua
vim.api.nvim_create_autocmd("InsertEnter", {
  group = vim.api.nvim_create_augroup("Disable_Caps", { clear = true }),
  desc = "Disable Caps in Insert mode",
  command = "silent !setxkbmap -option ctrl:nocaps",
})
```
